tools/scons/site_scons/site_tools/emscripten/emscripten.py

```diff
@@ -17,7 +25,7 @@ def generate(env, emscripten_path=None, **kw):
       raise 'Unable to find emscripten. Please set EMSCRIPTEN_ROOT'
```

Comment:
If `emscripten_path` is not currently set by default how does the current test in `test/scons/SConstruct` work?

Reply:
If I understand the question correctly, the current test uses the `@with_env_modify({'EMSCRIPTEN_ROOT': path_from_root()})` decorator to provide the environment variable used in line 15